ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/search/presentation/FoundOccurrenceNode.java

```diff
@@ -140,6 +140,7 @@ public boolean isLeaf() {
   //TODO
   public void updatePresentation(@NotNull NodePresentation presentation) {
     SpanElement spanElement = createSpanElement(styles.styles().presentableTextContainer());
+    spanElement.setAttribute("debugFilePath", itemPath);
```

Comment:
Actually you can use
```
spanElement.setId(UIObject.DEBUG_ID_PREFIX + "debug-file-path");
```

Reply:
```
spanElement.setId(UIObject.DEBUG_ID_PREFIX + "debug-file-path-" + itemPath);
```

Reply:
I used  attribute for span element, because there may be several cases found in file and the id is not suitable for this.

Reply:
Ok. use attribute then